cluster/gce/configure-vm.sh

```diff
@@ -851,6 +851,16 @@ function node-docker-opts() {
   if [[ -n "${EXTRA_DOCKER_OPTS-}" ]]; then
     DOCKER_OPTS="${DOCKER_OPTS:-} ${EXTRA_DOCKER_OPTS}"
   fi
+
+  # Decide whether to enable the cache
+  if [[ "${ENABLE_DOCKER_REGISTRY_CACHE:-}" == "true" ]]; then
+    REGION=$(echo "${ZONE}" | cut -f 1 -d -)
```

Comment:
@andyzheng0831 Added an explanation of the source of the variables (the kube-env metadata value), and added special handling for the ZONE variable being empty.

PTAL